src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -384,31 +384,32 @@ public CXmlResolverTest(ITestOutputHelper output) : base(output)
         [Theory]
         public void XmlResolver1(InputType inputType, ReaderType readerType, TransformType transformType, DocType docType)
         {
-            AppContext.SetSwitch("Switch.System.Xml.AllowDefaultResolver", true);
-
-            string expected = @"<?xml version=""1.0"" encoding=""utf-8""?><result><fruit>Apple</fruit><fruit>orange</fruit></result>";
-
-            try
+            using (new AllowDefaultResolverContext())
             {
-                if (LoadXSL("XmlResolver_Main.xsl", inputType, readerType) == 1)
+                string expected = @"<?xml version=""1.0"" encoding=""utf-8""?><result><fruit>Apple</fruit><fruit>orange</fruit></result>";
+
+                try
                 {
-                    xslt.XmlResolver = null;
-                    if (Transform("fruits.xml", transformType, docType) == 1)
+                    if (LoadXSL("XmlResolver_Main.xsl", inputType, readerType) == 1)
                     {
-                        VerifyResult(expected);
-                        return;
+                        xslt.XmlResolver = null;
+                        if (Transform("fruits.xml", transformType, docType) == 1)
+                        {
+                            VerifyResult(expected);
+                            return;
+                        }
+                        else
+                            Assert.True(false);
```

Comment:
For this and some other tests I wrapped their original code under an `using` statement to ensure that their change of the global context did not leak outside them - they were breaking some of the tests that I re-enabled. However, the diff here seems like a changed a bunch of their code... I will take a look at rewriting these tests avoiding these practices (`try-catch`, `Assert.True(false)`) but not I'm not sure if I will add it to this PR or later.